### ISSUE TITLE
Change the way harm stepping is handled (micro/macro)

### ISSUE THREAD

**Issue by Very-Soft** [VOREStation/VOREStation#2398]
Basically, I'd really prefer if actual harm couldn't be so easily accidentally done by walking down a hallway. The way it used to work was, in my opinion, better. Harm steps deliver halos damage. I'd even be fine with it dealing a lot of that, so they become basically disabled from it for a short time.

And move damage from harm stepping to a trait/ability, similar to how brutal predation works, but without the grab requirement. Give it a small cast time, and make it target a single person, similar to how Animal Nom works.

As is, there have been a number of times where some macro comes walking through a tight space on harm such as the tram or bar doors, and someone gets a broken bone. I've seen it happen to others, and it's happened to me a couple of times now.

I think that actual damage from that kind of thing should be more of a choice, and less abusable, as especially in the night hours, there's little to no medical to actually help with such things, and that's a pain. No pun intended.

**Comment by Dhaeleena** [comment]
The only way for a macro to deal great damage this way would be to be walking and also stepping on a micro with maximum size difference between each other. The damage is quite random, but it increases with the size difference between the two people. People should not be walking around in harm intent anyways and that something you can charge them with assault. I rarely have problems with people accidental harm stepping on me, Which is weird since i tend to play micros and have a foot/stomp fetish.
Evading the damage is also as easy as stepping out of the way of the 20ft dude when you are a mite and well charge them with assault for the fact. Just running over someone doesn't do as much damage as the walk.

**Comment by Very-Soft** [comment]
My issue with it is how easy it is to do damage to people by accident. If you're playing on hotkey mode and hit one of a couple of buttons and don't realize it, and then walk through a high traffic door, you very well may run over a micro you couldn't even see. 

I don't think that actual damage should be so easy to do by accident. Is why I think that damage via a usable ability would be better than making it tied to just walking over anyone smaller than you.

Just as an example, I play a small, not micro character, 75%-80%,  often fennec-ish things, so they tend to be small, weak, light things. So, with traits, they are fast, but pretty fragile. Then I run to the elevator and oops into some large person on harm and all of a sudden I'm in crit and have a broken leg.

I've saw someone a couple nights ago size ray themselves up and then walk out the door of the tram at night when there were no admins on, just stepping on people as they go. Probably should have Ahelped anyway for that, that's my bad. But I think like, harm from that should not be forced on others without active intent behind it.

It's easy to end up on hostile by mistake for one reason or another, be it sizegunning yourself for something, or trying to hotkey throw something and nudging one key off. 

I think it should be a more active choice, select an ability, that does harm to the person you targeted, similar to how brutal predation is an ability you have to use on someone. The present implementation of stomping has the potential for serious damage to other characters, and is easy to do by accident, and is easy to abuse. 

I think these things together make it a bit of a problem that's only going to lead people to getting in trouble. Some people will use it well and good, and that's great. But I think it could be done differently to be less abusable, in similar fashion to other things that exist presently and serve their purpose well.

**Comment by Very-Soft** [comment]
As an alternative, it could be an option in the vore panel like allow digestion

Allow foot crush?

If you have that disabled harm intent steps just do varying amounts of halos based on size difference. Enabling it allows damage.

I just really dislike the idea of people needing medical attention and potentially dying over something they aren't into or looking for, and the solution to that being to complain to security. Ultimately complaining to security is going to get the person a fine or a small jail sentence if they even stick around and are catchable. That's just getting a lot of people involved into something they probably aren't interested in to begin with over something small that may very well be a mistake.

When the way it used to work didn't really require people to come away from whatever they were doing to deal with it generally. I'm all for harm intent being used to disable tiny folk long enough to get them when they are doing crime and things, I think that's necessary, even for how robust a micro can be.

I just think that being hurt and potentially dying by that should not be able to be done by accident, is all. Not all of us who play smaller, weaker characters are into being crushed to death by some rando just passing by. (Though I do like that sometimes personally.) 

I in no way think that the option to damage/kill people with feet should be removed, I've just seen it done poorly to me and others a few times since I started playing a little more regularly again, and I thought it was a suggestion that could improve players options. Having options to opt into/out of super casual fetishy such things as this is in my opinion always for the better. The less ways one can accidentally murder someone who doesn't want to be murdered, the less people get accidentally in trouble.

**Comment by SilencedMP5A5** [comment]
speaking from experience, and im sure some people already know this

harm intent stomping is really really op.

for example:
- you can stunlock borgs for eternity as a macro.
- you can stunlock anyone you can trample, for eternity.
- you can KILL people SUPER easily by switching to harm intent and walking over them.

a macro can kill a micro faster than a shotgun can, probably, now that i think about it. : v
and as we all know from our many years of spacemans, stuns win fights.

**Comment by Screemonster** [comment]
Footstomping is logged now (or will be on the next release) but you absolutely should have ahelped that. We're also clarifying in space law that it's very much assault, and just like randomly running around toolboxing people, sizing yourself up and deliberately trampling people is an OOC issue.

Like seriously, someone sizes up and tramples the tram for the lulz and I see them doing it, they're sitting out a round for every person they step on.

**Comment by Very-Soft** [comment]
Yeah, as I said I recognize that I should have ahelped anyway about that. Will do so in the future. 

Additionally, was talking about this in discord, and came to understand that the reason this was swapped to damage, rather than halos is because certain people were going out of their way to antagonize those larger than them to get that as a reaction. Which I suppose makes sense as a deterrent, and potential RP tool.

But, as they checked and confirmed, 1 walk step from a macro on someone with low hp, (see:teshari, or very low endurance people) breaks a bone and puts them in crit. This is on non micro characters.

The tl;dr all my tired rambling above is, that's too high. I don't think that an accidental step should be a medical emergency. Finding a medkit or eating some donks should be enough to get you out of that, in case there's no medical, and an accident happens.

Save bone breaking, crit, and death for deliberate repeated steps.

**Comment by Treasa729** [comment]
I am very much for the idea of having a way to disable harm intent crushing. It's too easy to accidentally be on harm intent without realizing it.

**Comment by Cameron653** [comment]
I'll do some math this Saturday to make it much more balanced. 
Additionally, I'll be looking into making a toggle for prey to allow damage from being stepped on. If they allow damage via the toggle, they'll take brute damage. 
Otherwise, they'll take the damage as holodamage, multiplied by some number which will be worked out.
